config.tfvars

```diff
@@ -29,6 +43,28 @@ desired_capacity = 2
 #
 #domain = "<example.com>"
 
+# (Optional) URL for dataset to import
+# The provided default is the dataset used in the DCAPT framework.
+# See https://developer.atlassian.com/platform/marketplace/dc-apps-performance-toolkit-user-guide-bamboo/#2--preloading-your-bamboo-deployment-with-an-enterprise-scale-dataset
+# for details
+#
+#dataset_url = "https://centaurus-datasets.s3.amazonaws.com/bamboo/dcapt-bamboo.zip"
+
+
+################################################################################
```

Comment:
I think we can remove `Optional Configurations` title. All following items could go under Advanced.